workspaces/src/network/sandbox.rs

```diff
@@ -158,6 +162,70 @@ impl Sandbox {
         Ok(())
     }
 
+    pub(crate) async fn patch_account(
```

Comment:
Can you add tests for this and `patch_access_key`. Let me know if they aren't as straightforward to implement, but they should be simple if you follow the ones in [tests/patch_state.rs](https://github.com/near/workspaces-rs/blob/main/workspaces/tests/patch_state.rs).